modules/33acrossBidAdapter.js

```diff
@@ -569,21 +638,21 @@ function interpretResponse(serverResponse, bidRequest) {
 // All this assumes that only one bid is ever returned by ttx
 function _createBidResponse(response) {
```

Comment:
Optional change:
does it make sense to rename the param and use the term `bid` instead of `response` ?